brightener.h

```diff
@@ -1,12 +1,4 @@
 #pragma once
 #include "image.h"
 
-class ImageBrightener {
-private:
-	std::shared_ptr<Image> m_inputImage;
-public:
-	ImageBrightener(std::shared_ptr<Image> inputImage);
-	bool ValidateImage();
-	int BrightenWholeImage();
-	bool AddBrighteningImage(Image& imageToAdd, int& attenuatedCount);
-};
+int BrightenWholeImageWithLambda(std::shared_ptr<Image> image);
```

Comment:
Good to make this a simple function instead of a class with data member.
Consider keeping the function inside a namespace. e.g., if more such pixel-processing functions are needed